authorization.md

```diff
@@ -298,17 +300,21 @@ this, the service must first generate a one-time-use authentication JWT with the
 ```
 
 
-##### 2. Generate an RSA SHA-384 signed JWT over these claims
+##### 2. Digitally sign the claims, as specified in RFC7515.  
 
-Using the service's RSA private key, the signed token value is:
+Using the service's RSA private key, with SHA-384 hashing (as specified for 
+an `RS384` algorithm (`alg`) parameter value in RFC7518), the signed token 
+value is:
 
 ```
 eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzM4NCIsImtpZCI6ImVlZTlmMTdhM2I1OThmZDg2NDE3YTk4MGI1OTFmYmU2In0.eyJpc3MiOiJiaWxpX21vbml0b3IiLCJzdWIiOiJiaWxpX21vbml0b3IiLCJhdWQiOiJodHRwczovL2F1dGhvcml6ZS5zbWFydGhlYWx0aGl0Lm9yZy90b2tlbiIsImV4cCI6MTQyMjU2ODg2MCwianRpIjoicmFuZG9tLW5vbi1yZXVzYWJsZS1qd3QtaWQtMTIzIn0.l2E3-ThahEzJ_gaAK8sosc9uk1uhsISmJfwQOtooEcgUiqkdMFdAUE7sr8uJN0fTmTP9TUxssFEAQnCOF8QjkMXngEruIL190YVlwukGgv1wazsi_ptI9euWAf2AjOXaPFm6t629vzdznzVu08EWglG70l41697AXnFK8GUWSBf_8WHrcmFwLD_EpO_BWMoEIGDOOLGjYzOB_eN6abpUo4GCB9gX2-U8IGXAU8UG-axLb35qY7Mczwq9oxM9Z0_IcC8R8TJJQFQXzazo9YZmqts6qQ4pRlsfKpy9IzyLzyR9KZyKLZalBytwkr2lW7QU3tC-xPrf43jQFVKr07f9dA
+
 ```
 
-(Note: to inspect this example JWT, you can visit https://jwt.io, choose RS384,
-paste in the provided RSA keys, and then paste the JWT value into the "encoded"
-field.)
+
+Note: to inspect this example JWT, you can visit https://jwt.io. Paste the signed
+JWT value above into the "Encoded"  field, and paste the [sample public signing key](sample-jwks/RS384.public.json) (starting with the `{"kty": "RSA"` JSON object, and excluding the `{ "keys": [` JWK Set wrapping array) into the "Public Key" box.
+The plaintext JWT will be displayed in the "Decoded:Payload"  field, and a "Signature Verified" message will appear.
```

Comment:
@bakerdb I updated your PR to incorpore these more specific instructions for using jwt.io to inspect our sample JWT.